book/ch05.rst

```diff
@@ -262,7 +262,7 @@ For example, fig-tag-indian_ shows data accessed using
 
    POS-Tagged Data from Four Indian Languages: Bangla, Hindi, Marathi, and Telugu
 
-..    इराक_NNP के_PREP विदेश_NNC मंत्री_NN ने_PREP अमरीका_NNP के_PREP उस_PRP प्रस्ताव_NN का_PREP मजाक_NVB उड़ाया_VFM है_VAUX ...
+..    इराक_NNP के_PREP विदेश_NNC मंत्री_NN ने_PREP अमरीका_NNP के_PREP उस_PRP प्रस्ताव_NN का_PREP मजाक_NVB उड़ाया_VFM है_VAUX ...
 
```

Comment:
I'm not sure what changed but the diff on my machine didn't show this =(

Any ideas?